src/libraries/System.Private.CoreLib/src/System/Numerics/INumberBase.cs

```diff
@@ -191,9 +191,9 @@ static virtual TSelf CreateTruncating<TOther>(TOther value)
         /// </remarks>
         static abstract bool IsOddInteger(TSelf value);
 
-        /// <summary>Determines if a value is positive.</summary>
+        /// <summary>Determines if a value represents zero or a positive real number.</summary>
         /// <param name="value">The value to be checked.</param>
-        /// <returns><c>true</c> if <paramref name="value" /> is positive; otherwise, <c>false</c>.</returns>
+        /// <returns><c>true</c> if <paramref name="value" /> represents zero or a positive real number; otherwise, <c>false</c>.</returns>
```

Comment:
This isn't quite correct and potentially needs a clarification around the existence of `-0`. Likewise, the above docs for `IsNegative` would likewise need updating to clarify it returns `true` for `-0`.

Reply:
Good point; this needs to be clarified. I added a sentence about this in the remarks section; I personally don't think this detail needs to be very prominent because `-0` is not a commonly used value.